pipeline/outputs/elasticsearch.md

```diff
@@ -128,6 +143,46 @@ In your main configuration file append the following `Input` and `Output` sectio
 
 ![example configuration visualization from Calyptia](../../.gitbook/assets/image%20%282%29.png)
 
+### Configuration File with Upstream
+
+In your main configuration file append the following `Input` and `Output` sections:
+
+```text
```

Comment:
We need to provide yaml config as well now by default.

Reply:
Added in d3181511ea54b1b2f5a823c026530915cf857887